<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dkHastaListe</class>
 <widget class="QDialog" name="dkHastaListe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hastalar</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ADI</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>SOYADI</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CINSIYET</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>DOGUM TARIHI</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>TC KIMLIK</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>TELEFON</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ADRES</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>KAN GRUBU</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ALERJILER</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>KRONIK HASTALIKLAR</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ZIYARETLER</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblAra">
       <property name="text">
        <string>Ara</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leAra"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btnBilgi">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Bilgi</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnZiyaret">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Ziyaretler</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
